tor/strings/en_US.yml

```diff
@@ -324,10 +324,26 @@ formatting_issues:
   invalid_header: "**Invalid header**: It looks like your header doesn't contain a proper type (Image, Video, or Audio) or is missing italics. Please make sure your post contains a proper header. For example if it's an Image Transcription, it should be the following at the top of your transcription:
 
 
-    \   *Image Transcription: Example*
+    \    *Image Transcription: Example*
```

Comment:
At first I thought this was intentional, because yaml strings are incredibly frustrating to work with and very unintuitive, but it looks like it was indeed not rendering as a code block. Good catch!